src/microservices/Microservices.IdentifierMapper/Execution/Swappers/ForGuidIdentifierSwapper.cs

```diff
@@ -37,29 +38,36 @@ public ForGuidIdentifierSwapper()
         /// Connects to the specified swapping table if it exists, or creates it
         /// </summary>
         /// <param name="mappingTableOptions"></param>
-        public void Setup(IMappingTableOptions mappingTableOptions)
+        public override void Setup(IMappingTableOptions mappingTableOptions)
         {
             _options = mappingTableOptions;
             _table = _options.Discover();
 
-            CreateTableIfNotExists();
+            using(new TimeTracker(DatabaseStopwatch))
+                CreateTableIfNotExists();
         }
 
-        public string GetSubstitutionFor(string toSwap, out string reason)
+        public override string GetSubstitutionFor(string toSwap, out string reason)
         {
             reason = null;
 
             if (toSwap.Length > 10)
```

Comment:
Just noticed this. Looks like this specifically looks for CHIs (length 10), but some of the invalid PatientIDs we want to create a GUID for won't pass this check>

Reply:
Ok I'll check in a fix for this

Reply:
I have committed a fix for this, it now looks at the length of the column and enforces that as the maximum.